src/index.js

```diff
@@ -125,16 +132,44 @@ class JanusAdapter {
 
   async addOccupant(occupantId) {
     var subscriber = await this.createSubscriber(occupantId);
+
+    this.occupants[occupantId] = true;
+    this.occupantPeerConnections[occupantId] = subscriber.peerConnection;
+    this.audioStreams[occupantId] = subscriber.mediaStream;
+
+    // Resolve the promise for the user's media stream if it exists.
+    if (this.pendingAudioRequests[occupantId]) {
```

Comment:
These aren't just audio necessarily so we should call this `pendingMediaStreamRequests`